contribute/index.rst

```diff
@@ -1,11 +1,15 @@
 Contributing to UBports
 =======================
 
-Welcome! You're probably here because you want to contribute to UBports. The pages you'll find below here will help you do this in a way that's helpful to both the project and yourself.
+Welcome to the UBports Community! Joining our team, you will be part of a broad world effort by hundreds of people working to deliver a stunning Linux-friendly operating system. Meet new friends, learn new skills, and make a difference to our users while working with people from all around the globe.
 
-If you're just getting started, we always need help with :doc:`thorough bug reporting <bugreporting>`. If you are multilingual, :doc:`translations <translations>` are also a great place to start.
+Let's Get Started
+^^^^^^^^^^^^^^^^^
+
+We want to make sure that the UBports Community remains a welcoming and friendly place where people can feel comfortable. We ask you to abide by the UBports Community Code of Conduct when interacting with the rest of the UBports Community. Please make a note of our Code of Conduct at `Ubuntu.com/code-of-conduct <https://ubuntu.com/community/code-of-conduct>`_
 
-If those aren't enough for you, see `our contribute page <https://ubports.com/page/vo-get-involved>`_ for an introduction of our focus groups.
```

Comment:
seems to have moved to https://ubports.com/join-us

Reply:
That is UBports #1 dead end, nobody following this forum